README.md

```diff
@@ -1437,8 +1437,9 @@ Creates a snapshot of a volume within the data center. You can use a snapshot to
 |---|---|---|---|
 | dataCenterId | **yes** | string | The ID of the datacenter. |
 | volumeId | **yes** | string | The ID of the volume. |
-| name |  no | string | The name of the snapshot. |
+| name |  **yes** | string | The name of the snapshot. |
 | description | no | string | The description of the snapshot. |
+| licenceType | no | string | Set to one of the values: [WINDOWS, WINDOWS2016, LINUX, OTHER, UNKNOWN] |
```

Comment:
should we have "UNRECOGNIZED" type here or it is used only for technical purpose ?

Reply:
The only accepted values for the licenceType field accordingly Public API v5 documentation are: WINDOWS, WINDOWS2016, LINUX, OTHER, UNKNOWN. There is no UNRECOGNIZED value in the documentation.